api.md

```diff
@@ -469,7 +526,63 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
+
+Detach a data node from one hypertable or from all hypertables.
+
+Reasons for detaching a data node:
+
+- A data node should no longer be used by a hypertable and needs to be
+removed from all hypertables that use it
+- You want to have fewer data nodes for a distributed hypertable to
+partition across to other distributed hypertables in the database
+
+#### Required Arguments [](detach_data_node-required-arguments)
+
+| Name        | Description                       |
+|-------------|-----------------------------------|
+| `node_name` | Name of data node to detach from the hypertable |
+
+#### Optional Arguments [](detach_data_node-optional-arguments)
+
+| Name         | Description                            |
+|--------------|----------------------------------------|
+| `hypertable` | Name of the hypertable where the data node should be attached. If NULL, the data node will be detached from all hypertables. |
+| `force`      | Force detach of the data node even if that means that the replication factor is reduced below what was set. Note that it will never be allowed to reduce the replication factor below 1 since that would cause data loss.         |
+
+#### Returns
+
+| Column               | Description                              |
+|----------------------|------------------------------------------|
+| `hypertable_id`      | Hypertable id of the modified hypertable |
+| `node_hypertable_id` | Hypertable id on the remote data node    |
+| `node_name`          | Name of the attached data node     |
 
+#### Errors
+
+Detaching a node is not permitted:
+- If it would result in data loss for the hypertable due to the data node
+containing chunks that are not replicated on other data nodes
+- If it would result in under-replicated chunks for the distributed hypertable
```

Comment:
Maybe add note that replication is currently experimental or simply unsupported.

Reply:
Done